compiler/cpp/src/thrift/generate/t_js_generator.cc

```diff
@@ -860,14 +883,15 @@ void t_js_generator::generate_js_struct_reader(ostream& out, t_struct* tstruct)
   indent(out) << "input.readStructBegin();" << endl;
 
   // Loop over reading in fields
-  indent(out) << "while (true)" << endl;
+  indent(out) << "while (true) {" << endl;
 
-  scope_up(out);
+  indent_up();
 
-  indent(out) << "var ret = input.readFieldBegin();" << endl;
-  indent(out) << "var fname = ret.fname;" << endl;
```

Comment:
Is fname no longer important?

Reply:
I've removed instances of temporary variables that are not used in any of the generated code. There is currently a lot of clutter right now.